dev/com.ibm.websphere.appserver.features/visibility/public/jaxws-2.3/com.ibm.websphere.appserver.jaxws-2.3.feature

```diff
@@ -5,6 +5,7 @@ singleton=true
 IBM-App-ForceRestart: uninstall, \
```

Comment:
Initially, that was me following the jaxrs-2.1 model a bit, and I would have agreed that it is unnecessary in retrospect. I have another set of changes built on top of this for the client container runtime update, and I actually refactored the jaxws-2.3 features, and put all of the common dependencies between the client and server side containers into a private jaxws.common.2.3 feature. Would it be OK if I committed as is, and then moved the non-common dependencies to the public feature as a part of my refactor for the client container?

Reply:
Yeah if more than one feature depended on the internal feature it would make sense to have one.  Since you have a plan moving forward to have a common private feature for both jaxws and jaxwsclient, I am fine with leaving it the way it is.  I still think you should have dependency on eeCompatible-8.0 though.